docs/Gemfile

```diff
@@ -0,0 +1,34 @@
+# © 2020 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+
+source "https://rubygems.org"
+# Hello! This is where you manage which Jekyll version is used to run.
+# When you want to use a different version, change it below, save the
+# file and run `bundle install`. Run Jekyll with `bundle exec`, like so:
+#
+#     bundle exec jekyll serve
+#
+# This will help ensure the proper Jekyll version is running.
+# Happy Jekylling!
+
```

Comment:
Sorry to quibble, but by *removing* the jekyll line here, the "This will help ensure..." comment above is orphaned. Please put the line back but commented out.

And then below say "uncomment the line" not "add the line".

Reply:
Good point -- I'll force-push an update to change this.